.gitignore

```diff
@@ -133,4 +133,8 @@ dmypy.json
 # Pyre type checker
 .pyre/
 
-.DS_Store
\ No newline at end of file
+# docker-compose yml
```

Comment:
can you get rid of this extra line please?

Reply:
Sure, I have removed the comment